libs/blocks/library-config/library-config.js

```diff
@@ -143,8 +143,7 @@ async function combineLibraries(base, supplied) {
 function createList(libraries) {
   const container = createTag('div', { class: 'con-container' });
 
-  const libraryList = createTag('ul', { class: 'sk-library-list' });
-  container.append(libraryList);
+  const libraryList = createTag('ul', { class: 'sk-library-list' }, null, { parent: container });
```

Comment:
you missed one element here - https://github.com/adobecom/milo/blob/694c4d198e7212481beab638d0d6e61d3122ac2d/libs/blocks/library-config/library-config.js#L151